diesel/src/mysql/connection/mod.rs

```diff
@@ -130,7 +130,7 @@ mod tests {
     use std::env;
 
     fn connection() -> MysqlConnection {
-        let _ = dotenv::dotenv();
```

Comment:
Should I keep this? I think it's better to use `.ok()` to keep consistency for now.